src/main/java/WayofTime/alchemicalWizardry/common/rituals/RitualEffectBiomeChanger.java

```diff
@@ -75,61 +80,38 @@ public void performEffect(IMasterRitualStone ritualStone) {
             }
 
             boolList[range][range] = true;
-            boolean isReady = false;
 
-            while (!isReady) {
-                isReady = true;
+            Queue<Integer> BFSqueue = new ArrayDeque<>();
```

Comment:
Some algorithmic explanation would be greatly appreciated.

Reply:
It's just a breadth-first search to find valid coordinates to change. (One y layer above the master ritual stone, bounded by bloodstone bricks or maximum distance.) I'm not sure what else I should explain?